packages/ckeditor5-image/src/imageupload/uploadimagecommand.ts

```diff
@@ -21,7 +21,7 @@ import type ImageUtils from '../imageutils.js';
  * and it is also available via aliased `imageUpload` name.
  *
  * In order to upload an image at the current selection position
- * (according to the {@link module:widget/utils~findOptimalInsertionRange} algorithm),
+ * (according to the {@link module:engine/model/schema~Schema#findOptimalInsertionRange} algorithm),
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa